fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg

```
@@ -0,0 +1,63 @@
+[aircraft]
+icaotype = A20N
+parkingbrakestest = (L:A32NX_PARK_BRAKE_LEVER_POS,number)
+nosegear = 8.26
+refueling = 0
+battery = 1
+pushbackraise = 1
+pushbackdummyevent = 0
+pushbackcheckengines = 0
+iscargo = 0
+trafficcones = 1
+preferredexit =  0
+wingrootpos = 2.36 -1.89 0.85
+wingtippos = 15.70 -7.41 1.84
+fuelpos = 9.63 -3.39 0.88
+waterpos = 1.28 -18.07 0.52
+lavatorypos = 0.59 -18.07 0.12
+gpupos = 3.29 11.06 -1.56
+bypasspinpos = 0.08 8.45 -1.42
+engine1pos = -5.67 0.50 -0.81
+engine2pos = 5.67 0.50 -0.80
+engine3pos = 0.00 0.00 0.00
+engine4pos = 0.00 0.00 0.00
+
+[exit1]
+pos = -1.88 8.51 0.86 4.00
+code = 1
+name = Passenger Door
+embeddedStair = 0
+
+[exit2]
+remove = 1
+
+[exit3]
+remove = 1
+
+[exit4]
+remove = 1
+
+[service1]
+pos = 2.01 8.50 0.99 -5.00
+code = 4
+name = Service Door
+embeddedStair = 0
```

As in the exp integration.
```suggestion
[service1]
remove = 1
```